pyhttp/Exceptions.py,sha256=q3V4WYYffQu4_EQk1NLy8amvcDHVVnYGJeMkJ1XjQ3w,611
pyhttp/Request.py,sha256=Oa8unyixOu0LDSATMdKi3at4WSBRznjbmfxDKFAgjOM,2264
pyhttp/Response.py,sha256=BljFeVG5QVKd-WZwzH4GxNzxiXaa3t9TVWBIl5VaeiY,621
pyhttp/__init__.py,sha256=RC0FaW2w7gzL_k9RNvrTePmMN_8UVqayOcBsabIbqe8,101
pyhttp/__pycache__/Exceptions.cpython-38.pyc,,
pyhttp/__pycache__/Request.cpython-38.pyc,,
pyhttp/__pycache__/Response.cpython-38.pyc,,
pyhttp/__pycache__/__init__.cpython-38.pyc,,
pyrequest-0.5.dist-info/INSTALLER,sha256=zuuue4knoyJ-UwPPXg8fezS7VCrXJQrAP7zeNuwvFQg,4
pyrequest-0.5.dist-info/METADATA,sha256=zHX9p0bHKb6nQzx9Eo1HKmunT_zhKFBwkTdDOhI3uHc,1775
pyrequest-0.5.dist-info/RECORD,,
pyrequest-0.5.dist-info/WHEEL,sha256=g4nMs7d-Xl9-xC9XovUrsDHGXt-FT0E17Yqo92DEfvY,92
pyrequest-0.5.dist-info/top_level.txt,sha256=CNdrskwb-a5eLXkPyQ3TztsV_AOpwIMYWRz0lcQGz2M,7
